docs/configuration/backends/kubernetes.md

```diff
@@ -43,6 +43,13 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 #
 # namespaces = ["default", "production"]
 
+# Namespace label selector to filter namespaces that should be watched.
+#
+# Optional
+# Default: empty (watch all namespaces).
+#
+# namespaceLabelSelector = "label = value"
```

Comment:
We should document the behavior if both this new option and `namespaces` are defined.